packages/webapp/src/containers/WeatherBoard/index.jsx

```diff
@@ -29,7 +20,7 @@ export default function WeatherBoard() {
   const formattedForecast = {
     humidity: `${t('WEATHER.HUMIDITY')}: ${humidity}`,
     iconName,
-    date: utils.formatDate(language_preference, date),
+    date: utils.formatDate(language_preference, date ? date * 1000 : new Date()),
```

Comment:
I think one of these two are throwing a new warning from moment as I don't see it on the base branch.

![Screenshot 2024-09-24 at 11 16 19 AM](https://github.com/user-attachments/assets/3602a086-2900-4370-9b82-7f1556f6af64)

Reply:
This `formatDate` function doesn't use moment so it shouldn't be the source of this warning, right? I'm not using moment at any point in this PR -- we should look into that warning though!

Reply:
Right wrong spot! 

I think our `util/moment.js` file is where the new Intl.DateTime format interact with moment then maybe:

![Screenshot 2024-09-25 at 10 44 27 AM](https://github.com/user-attachments/assets/542a355f-b488-48cd-8cfd-e538645f9c0d)

I didn't see this warning before on base branch to this but .. it is just a warning.